compiler/rustc_serialize/src/opaque.rs

```diff
@@ -17,6 +17,8 @@ use crate::int_overflow::DebugStrictAdd;
 
 pub type FileEncodeResult = Result<usize, (PathBuf, io::Error)>;
 
+const FOOTER: &[u8] = b"rust-end-file";
```

Comment:
I feel like naming this *footer* is a bit confusing since we already have the concept of *file footers*.

Reply:
I changed the name to something like "magic bytes, but at the end".

Reply:
Haha, that does describe it pretty accurately.